### ISSUE TITLE
k-mer repeat count Error

### ISSUE THREAD

**Issue by patidarr** [OSU-SRLab/MANTIS#10]
Hi,
I got following error:
`
Microsatellite Analysis for Normal-Tumor InStability (v1.0.3)
python /gpfs/gsfs6/users/MoCha/patidarr/MANTIS/kmer_repeat_counter.py \
-b /data/MoCha/patidarr/MANTIS/loci.bed \
-n /gpfs/gsfs6/users/MoCha/processedDATA/714841/20170910/714841_germline~WES/714841_germline~WES.bwa.final.bam \
-t /gpfs/gsfs6/users/MoCha/processedDATA/714841/20170910/714841~288-R~KV6MF7NX7~WES/714841~288-R~KV6MF7NX7~WES.bwa.final.bam \
-o /gpfs/gsfs6/users/MoCha/processedDATA/714841/20170910/714841~288-R~KV6MF7NX7~WES/MANTIS/714841~288-R~KV6MF7NX7~WES.kmer_counts.txt \
--min-read-quality 25.0 \
--min-locus-quality 30.0 \
--min-read-length 35 \
--genome /data/MoCha/patidarr/ref/ucsc.hg19.fasta \
--threads 20
Getting repeat counts for repeat units (k-mers) ...
Error with k-mer repeat count calculations; terminating program.
`

Could you please let me know how do I resolve it?

Thanks,
Rajesh

**Comment by rbonneville** [comment]
Is kmer_repeat_counter.py in the same folder as mantis.py?

**Comment by patidarr** [comment]
Yes they are in the same folder. I got error on a few samples of >1000 I processed.

**Comment by rbonneville** [comment]
Try running the new [iss010](/OSU-SRLab/MANTIS/tree/iss010) branch. This won't fix the problem, but should provide the information needed to identify it.

**Comment by patidarr** [comment]
Sure, I can try the new branch.

Thanks!!
